protos-frontend/frontend/FrontendService.proto

```diff
@@ -34,5 +36,35 @@ message TriggerPhoneVerificationRequest {
 }
 
 message TriggerPhoneVerificationResponse {
-    enums.PhoneNumberVerificationResponseType response_type = 1;
+    enums.VerificationResponseType response_type = 1;
 }
+
+
+
+message GetEmailVerificationScreenRequest {
+    string img_url = 1;
+    string title = 2;
+    string sub_title = 3;
+    string frame = 4;
+    string your_emailid = 5;
+    string your_name = 6;
+    ui.Button next_button = 7;
+}
+
+message GetEmailVerificationScreenResponse {
+    enums.VerificationResponseType response_type = 1;
+    string img_url = 2;
+    string title = 3;
+    string sub_title = 4;
+    string frame = 5;
+    string your_emailid = 6;
+    string your_name = 7;
+    ui.Button next_button = 8;
+}
+
+message TriggerEmailVerificationRequest {
+    string email_id = 1;
```

Comment:
you need to send both parameters from User right? else there is no point collecting it in the first place